studio-docs/source/explorer/connecting-authenticating.mdx

```diff
@@ -167,6 +167,21 @@ Function that prompts the user for input and returns the value in a promise. If
 </td>
 </tr>
 
+<tr>
+<td>
+
+##### `explorer.request.body`
+
+`
+{ query: string; variables: { [key in string]?: JSONValue } | null; operationName: string | undefined; }
+`
+</td>
+<td>
+
+The body of the `POST` request that will be sent to the configured graphql endpoint in the current request.
```

Comment:
Recommend: "The body of the `POST` request that's sent to the configured GraphQL endpoint for the current operation."